python/cudf/cudf/core/frame.py

```diff
@@ -1066,14 +1066,22 @@ def _copy_categories(self, other, include_index=True):
                     offset=col.offset,
                 )
         if include_index:
-            from cudf.core.index import RangeIndex
+            from cudf.core.index import as_index, CategoricalIndex, RangeIndex
```

Comment:
Get rid of these imports within the function, they're actually somewhat expensive and just use fully qualified names within here. i.e. `cudf.core.index.RangeIndex`, `cudf.core.index.as_index`, etc.